Dockerfile

```diff
@@ -1,14 +1,12 @@
-FROM --platform=$BUILDPLATFORM alpine AS builder
-
-FROM scratch
-COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
-
+FROM cgr.dev/chainguard/static:latest-20230102
 # Normally we would set this to run as "nobody", but to play nicely with GitHub
 # Actions, it must run as the default user:
 #
 #   https://docs.github.com/en/actions/creating-actions/dockerfile-support-for-github-actions#user
 #
 # USER nobody
+WORKDIR /ws
```

Comment:
I don't see the value in hard coding this. Users can boot the image with "-w" to set a working directory if needed.

Reply:
It gives a much better out of the box experience. Of course users can customize anything they desire, but this gives at least out of the box an easy experience without customizations.

Reply:
I don't agree. This relies on the user being willing to recognize and remember a random directory name (`/ws`) versus being explicit. I'm fine updating the README with an explicit example for running the container as a binary, but the rest of these changes seem unnecessary:

```shell
docker run -it --rm -v $PWD:$PWD -w $PWD ghcr.io/sethvargo/ratchet:0.4.0 "$@"
```